src/AstroImages.jl

```diff
@@ -92,17 +106,19 @@ function AstroImage(file::String)
 end
 
 # Lazily render the image as a Matrix{Color}, upon request.
-function render(img::AstroImage{T,C}) where {T,C}
-    imgmin, imgmax = extrema(img.data)
+function render(img::AstroImage{T,C,N}, header_number = 1) where {T,C,N}
+    imgmin, imgmax = extrema(img.data[header_number])
     # Add one to maximum to work around this issue:
     # https://github.com/JuliaMath/FixedPointNumbers.jl/issues/102
     f = scaleminmax(_float(imgmin), _float(max(imgmax, imgmax + one(T))))
-    return colorview(C, f.(_float.(img.data)))
+    return colorview(C, f.(_float.(img.data[header_number])))
 end
 
 Images.colorview(img::AstroImage) = render(img)
 
-Base.size(img::AstroImage) = Base.size(img.data)
+Base.size(img::AstroImage) = Base.size.(img.data)
```

Comment:
Do you have a concrete case where this definition would be useful?  I'm not sure that anyone would expect the standard `size` to return a tuple of tuples.  For the case of the single image I think this function could have been, but now in this general case I'm a bit sceptical.